etc/trunion-dev.ini

```diff
@@ -80,3 +80,23 @@ certfile = trunion/tests/test_crt.jwk
 chainfile = trunion/tests/test_x509_chain.pem
 permitted_issuers = https://marketplace-dev.allizom.org
 we_are_signing = receipts
+
+[addons]
+ca_cert_file = /etc/trunion/addons_root_ca_cert.pem
```

Comment:
is this going to break if you merge before this file exists? If so, please file an [ops bug](https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla%20Services&component=Operations:%20Marketplace) to get it in place. Or sometimes you can just do it on IRC.